packages/core/src/controllers/user.ts

```diff
@@ -84,24 +82,30 @@ export class UserController extends Controller {
 
       const getWalletAddress = async (): Promise<string | null> => {
         if (isUserID) {
-          const [user] = await this.getUserByID({ userID: decodedClaims.sub });
+          const { data: user } = await this.getUserByID({
+            userID: decodedClaims.sub,
+          });
 
-          return user && user.wallet;
+          return (user && user.wallet) || null;
```

Comment:
Should this be a 404 if it's not found?

Reply:
hmm, I think null makes more sense here

we could return another `ResponseData` if we wanted to return statusCode but that seems like overkill